src/execute/hac_sys-pcode-interpreter-composite_data.adb

```diff
@@ -84,11 +83,7 @@ package body HAC_Sys.PCode.Interpreter.Composite_Data is
       H2 := Index (ND.S (Curr_TCB.T).I);      --  Index to string table
       H3 := Index (IR.Y);                     --  Size of array
       H4 := Index (ND.S (Curr_TCB.T - 1).I);  --  Length of string
-      if H3 < H4 then
-        H5 := H1 + H3;    --  H5 is H1 + min of H3, H4
-      else
-        H5 := H1 + H4;
-      end if;
+      H5 := H1 + (if H3 < H4 then H3 else H4);
```

Comment:
Comment --  H5 is H1 + min of H3, H4
seems no longer needed ;-)

Reply:
Mmmh... not sure what you mean. What is no longer needed?

Reply:
Since, I consider the comment in
```
H5 := H1 + (if H3 < H4 then H3 else H4);      -- H5 is H1 + min of H3, H4
```
not adding much value.

Reply:
Comment was originally at line 88 (and not repeated at 90)

Reply:
Got it! I thought that you made a comment that H5 is no longer needed :-)

Reply:
Changed - much nicer indeed!